tests/PaymentBundle/Controller/PaymentControllerTest.php

```diff
@@ -0,0 +1,186 @@
+        $this->assertSame($basket, $methodGetBasket->invoke($paymentController));
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testInstanceNullBasketInterfacePaymentAction()
+    {
+        $classPaymentController = new \ReflectionClass(PaymentController::class);
+        $methodGetBasketFactory = $classPaymentController->getMethod('getBasketFactory');
+        $methodGetBasketFactory->setAccessible(true);
+
+        $methodGetBasket = $classPaymentController->getMethod('getBasket');
+        $methodGetBasket->setAccessible(true);
+
+        $methodGetPaymentHandler = $classPaymentController->getMethod('getPaymentHandler');
+        $methodGetPaymentHandler->setAccessible(true);
+
+        $basketFactoryInterface = $this->createMock(BasketFactoryInterface::class);
+        $basket = $this->createMock(Basket::class);
+        $paymentHandlerInterface = $this->createMock(PaymentHandlerInterface::class);
+
+        $container = $this->createMock(ContainerInterface::class);
+        $container->expects($this->at(0))
+            ->method('get')
+            ->with('sonata.basket.factory')
+            ->willReturn($basketFactoryInterface);
+
+        $container->expects($this->at(1))
+            ->method('get')
+            ->with('sonata.basket')
+            ->willReturn($basket);
+
+        $container->expects($this->at(2))
+            ->method('get')
+            ->with('sonata.payment.handler')
+            ->willReturn($paymentHandlerInterface);
+
+        $paymentController = new PaymentController();
+
+        $paymentController->setContainer($container);
+
+        $this->assertSame($basketFactoryInterface, $methodGetBasketFactory->invoke($paymentController));
+        $this->assertSame($basket, $methodGetBasket->invoke($paymentController));
+        $this->assertSame($paymentHandlerInterface, $methodGetPaymentHandler->invoke($paymentController));
+    }
+
+    public function testSendBankAction()
+    {
+        $request = new Request();
+        $request->setMethod('POST');
+
+        $payment = $this->getDebugPayment();
+
+        $order = $this->getOrder();
+
+        $paymentHandlerInterface = $this->createMock(PaymentHandlerInterface::class);
+        $paymentHandlerInterface->expects($this->once())
+            ->method('getSendbankOrder')
+            ->willReturn($order);
+
+        $basket = $this->createMock(Basket::class);
+        $basket->expects($this->once())
+            ->method('isValid')
+            ->willReturn(true);
+
+        $basket->expects($this->once())
+            ->method('getPaymentMethod')
+            ->willReturn($payment);
+
+        $paymentController = $this->createPaymentController(null, $paymentHandlerInterface, $basket);
+
+        $this->assertInstanceOf(Response::class, $paymentController->sendbankAction($request));
+    }
+
+    /**
+     * @return \Sonata\Component\Order\OrderInterface
+     */
+    protected function getOrder()
```

Comment:
Please remove the phpdoc, add a use statement and  a return type declaration:

```suggestion
    private function getOrder(): OrderInterface
```